src/utils/locale.ts

```diff
@@ -52,11 +51,12 @@ export const locales = [
   "zh-TW"
 ];
 
-function createLocaleNumberFormatter(locale: string): Intl.NumberFormat {
+function createLocaleNumberFormatter(locale: string, numberingSystem?: string): Intl.NumberFormat {
```

Comment:
You could also set `latn` as the default option in the `numberingSystem` parameter.

Reply:
The browser preference is set as default in the `input` component .

Reply:
I was thinking of just having the default in here in case this gets used in other places where a default might not exist or is not needed, so it would look like: `function createLocaleNumberFormatter(locale: string, numberingSystem: string = 
 "latn"): Intl.NumberFormat`